src/scala/annotation/static.scala

```diff
@@ -0,0 +1,13 @@
+package scala.annotation
+
+import scala.annotation.meta._
+
+/** https://github.com/scala/scala.github.com/pull/491 */
+
+@field
+@getter
+@beanGetter
+@beanSetter
+@param
```

Comment:
I guess the question is: what *is* a static param, even if produced by a tool/optimizer/analyzer? I don't have any idea what that would look like in the bytecode.